src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {

```

Comment:
class Ball is a model and shouldn't extend service Lottery
```suggestion
public class Ball {
```